### PR TITLE
Fix office filter on admin dashboard

### PR DESCRIPTION
**Pull request by danielbreves**
Admin dashboard was throwing: `Uncaught TypeError: Cannot read property 'offices' of undefined`

cc @zendesk/volunteer

### CODE REVIEW

File: app/javascript/pages/admin/Dashboard/index.js [hunk 1]
```diff
@@ -78,8 +78,8 @@ const Dashboard = props => {
   const spotsThisWeek = R.mapAccum(spotsAppender, 0, eventsThisWeek)[0]
   const spotsThisMonth = R.mapAccum(spotsAppender, 0, eventsThisMonth)[0]
 
-  const spotsFilledThisWeek = spotsThisWeek > 0 ? Math.round(volunteersThisWeek / spotsThisWeek * 100) : 0
-  const spotsFilledThisMonth = spotsThisMonth > 0 ? Math.round(volunteersThisMonth / spotsThisMonth * 100) : 0
+  const spotsFilledThisWeek = spotsThisWeek > 0 ? Math.round((volunteersThisWeek / spotsThisWeek) * 100) : 0
+  const spotsFilledThisMonth = spotsThisMonth > 0 ? Math.round((volunteersThisMonth / spotsThisMonth) * 100) : 0
```
Comment (craig-day) [inline app/javascript/pages/admin/Dashboard/index.js]: I'm pretty sure that prettier.js was removing these for me. Did you add them, or did it?

File: app/javascript/pages/admin/Dashboard/index.js [hunk 2]
```diff
@@ -173,6 +170,9 @@ const mapStateToProps = (state, _ownProps) => ({
   adminOfficeFilter: state.model.adminOfficeFilter,
 })
 
-const withActions = connect(mapStateToProps, {})
+const withActions = connect(
+  mapStateToProps,
+  {}
+)
```
Comment (craig-day) [inline app/javascript/pages/admin/Dashboard/index.js]: prettier or you?

**Review by craig-day — APPROVED** [review]
Assuming the style changes were from prettier, and not you, looks good!

**Comment by danielbreves** [comment]
Yea, the style changes were from prettier. Thanks!

**Pull request merged**